migrations/20190313132436_exercisesNew.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.hasTable('exercises').then(function(exists) {
+    if (!exists) {
+      return knex.schema.createTable('exercises', function(table) {
```

Comment:
If you want to alter the table, you should use knex.schema.table. This won't work since the table aleady exists.